RHDEVDOCS-5315 - Logging 5.7.1 Release Notes (#60138)
openshift/openshift-docs: modules/cluster-logging-rn-5.7.1.adoc

//module included in cluster-logging-release-notes.adoc
:_content-type: REFERENCE
[id="cluster-logging-release-notes-5-7-1_{context}"]
= Logging 5.7.1
This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].

> [id="openshift-logging-5-7-1-bug-fixes"]

Reviewer: all module IDs need to include context per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files

== Bug fixes

> * Before this update, the presence of numerous noisy messages within Cluster Logging Operator pod logs caused reduced log readability and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs, resulting in improved log readability and enhanced visibility of critical system events. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])

Reviewer: ```suggestion
* Before this update, the presence of numerous noisy messages within the Cluster Logging Operator pod logs caused reduced log readability, and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
```

suggested edit for readability - added a comma, and I don't think the last line needs to be there since it's already mentioned previously as the result of having too many messages

Reviewer: Have you considered also adding an attribute for Cluster Logging Operator? Not important for this PR since it's maybe out of scope, but worth considering opening a follow up issue IMO.

> * Before this update, the API server would default the new collector type field to "vector" even when the custom resource had the prior field selected. This update removes the default for the new collector type field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])

Reviewer: ```suggestion
* Before this update, the API server would reset the value for the `CollectorSpec.Type` field to `vector`, even when the custom resource used a different value. This update removes the default for the `CollectorSpec.Type` field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])
```

Purely a suggestion, but if there's a literal name here that can be used for the field to clarify what is being referred to, I'd go with that instead of "new collector type field".

Also, put literal values in `backticks` instead of "quotation marks" per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

> * Before this update, within the {Product-Title} web console a time range could not be selected by click dragging over the logs histogram. With this update, click dragging can successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])

Reviewer 2: Wrong bug link. it should be https://issues.redhat.com/browse/LOG-4051

Reviewer 2: @libander I still see the wrong bug link

> * Before this update, within the {Product-Title} web console a time range could not be selected by click dragging over the logs histogram. With this update, click dragging can successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])

Reviewer: ```suggestion
* Before this update, a time range could not be selected in the {Product-Title} web console by clicking and dragging over the logs histogram. With this update, clicking and dragging can be used to successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
```

suggested editing for clarity

Reviewer: not positive wrt. "click dragging" as a term, but per https://www.ibm.com/docs/en/ibm-style?topic=word-usage I'd probably spell it out like this to be safe - might be worth asking core OCP team for advice or looking for other instances in the docs. By doing a quick search, it looking like "clicking and dragging" is used in other places, e.g. https://docs.openshift.com/container-platform/4.13/monitoring/querying-metrics.html#exploring-the-visualized-metrics_querying-metrics

> * Before this update, the action of clicking on the *Show Resources* link within the {Product-Title} web console did not produce any effect. With this update, the issue is resolved by fixing the functionality of the "Show Resources" link to toggle the display of resources for each log entry. (link:https://issues.redhat.com/browse/LOG-3218[LOG-3218])

Reviewer: ```suggestion
* Before this update, clicking on the *Show Resources* link in the {Product-Title} web console did not produce any effect. With this update, the issue is resolved by fixing the functionality of the "Show Resources" link to toggle the display of resources for each log entry. (link:https://issues.redhat.com/browse/LOG-3218[LOG-3218])
```

suggested editing for minimalism

> [id="openshift-logging-5-7-1-CVEs"]

Reviewer: ```suggestion
[id="openshift-logging-5-7-1-CVEs_{context}"]
```

all module IDs need to include context per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files
